tests/foreman/destructive/test_puppetplugin.py

```diff
@@ -95,9 +94,7 @@ def test_positive_enable_disable_logic(target_sat, capsule_configured):
 
     # Enable puppet on Satellite and check it succeeded.
     target_sat.register_to_cdn()
-    result = target_sat.execute(enable_satellite_cmd.get_command(), timeout='20m')
-    assert result.status == 0
-    assert 'Success!' in result.stdout
+    target_sat.enable_puppet()
```

Comment:
This is not needed as the puppet is already enabled, this is running the installer command which we consider as a destructive method. 

If we still want to run puppet enablement, we should move testing to destructive sat.

Reply:
For now, we could go with `without` enabling satellite and when scenario demands for it OR if we change the templates to disable puppet by default, we then think about this use.

Please remove this line.